src/main/java/org/togetherjava/discordbot/db/repository/SimpleRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.togetherjava.discordbot.db.repository;
+
+import org.jooq.*;
+import org.jooq.impl.DSL;
+import org.togetherjava.discordbot.commands.CommandContext;
+
+public abstract class SimpleRepository<T> implements Repository<T> {
```

Comment:
Since this klass seems to be a baseclass of Jooq implementations. I'd rename this class so that the name reflects what this class represents.